<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oslab</class>
 <widget class="QMainWindow" name="oslab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>oslab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新建进程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>131</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Group1 (名称/时间)</string>
    </property>
    <widget class="QTextBrowser" name="group1name1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1time1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1name2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1name3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1name4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1time2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1time3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="group1time4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>131</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Group 2  (名称/时间)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>131</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Group 3  (名称/时间)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>stopButton</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
